lib/rubocop/cop/rails/dynamic_find_by.rb

```diff
@@ -57,6 +61,33 @@ def autocorrect(node)
 
         private
 
+        def allowed?(node)
+          allowed_method?(node) || allowed_receiver?(node) ||
+            whitelisted?(node)
+        end
+
+        def allowed_method?(node)
+          return unless cop_config['AllowedMethods']
+
+          cop_config['AllowedMethods'].include?(node.method_name.to_s)
+        end
+
+        def allowed_receiver?(node)
+          return unless cop_config['AllowedReceivers'] && node.receiver
+
+          cop_config['AllowedReceivers'].include?(node.receiver.source.to_s)
```

Comment:
`RuboCop::AST::Node#source` returns String type, `to_s` looks unnecessary.

```suggestion
          cop_config['AllowedReceivers'].include?(node.receiver.source)
```